packages/solarflare-theme/package.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "solarflare-theme",
+	"displayName": "Solarflare Theme",
+	"version": "0.0.1",
+	"description": "The official VSCode theme extension for Cloudflare.",
+	"categories": [
+		"Themes"
+	],
+	"keywords": [
+		"wrangler",
+		"cloudflare",
+		"workers",
+		"cloudflare workers",
+		"vscode",
+		"theme",
+		"cloudflare theme"
+	],
+	"homepage": "https://github.com/cloudflare/workers-sdk/packages/vscode-theme#readme",
+	"bugs": {
+		"url": "https://github.com/cloudflare/workers-sdk/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/cloudflare/workers-sdk.git",
+		"directory": "packages/vscode-theme"
+	},
+	"license": "MIT OR Apache-2.0",
+	"author": "wrangler@cloudflare.com",
```

Comment:
Should this be workers-devprod@?

Reply:
we should probably update all the packages in the monorepo then.